recipe/install-core.sh

```diff
@@ -22,24 +21,19 @@ popd
 echo "done"
 set -e
 
-cp -R * "$INSTALL_DIR"
-rm -rf "$INSTALL_DIR/obj"
-rm -rf "$INSTALL_DIR/pkg"
-rm "$INSTALL_DIR"/conda_build.sh
-cp "$RECIPE_DIR/gap.sh" "$PREFIX/bin/gap"
-chmod +x "$PREFIX/bin/gap"
+mkdir -p "$INSTALL_DIR"/pkg
+cp -R pkg/* "$INSTALL_DIR"/pkg
 
-GAP_SRC_PATH=`ls -d "$INSTALL_DIR"/bin/*/src`
-rm "$GAP_SRC_PATH"
-ln -s "$INSTALL_DIR"/src/ "$GAP_SRC_PATH"
+mkdir -p "$INSTALL_DIR"/src
+cp -R src/* "$INSTALL_DIR"/src
 
 mkdir -p "$INSTALL_DIR/pkg"
 cd pkg
 for GAP_PKG_NAME in smallgrp transgrp primgrp gapdoc;
 do
-    PKG_DIR=`find . -maxdepth 1 -iname "$GAP_PKG_NAME-*" -o -iname "$GAP_PKG_NAME" -type d`
     echo "GAP_PKG_NAME: $GAP_PKG_NAME"
-    echo "PKG_DIR: $PKG_DIR"
-    ls -al .
-    mv $PKG_DIR $INSTALL_DIR/pkg/$PKG_DIR
+    mv $GAP_PKG_NAME $INSTALL_DIR/pkg/$GAP_PKG_NAME
 done
+
+# Delete doc for agt (we do not ship docs and these contain broken symlinks.)
```

Comment:
Ok. Let's look into this in a separate issue. There's is usually no point in shipping PDFs and such as most people wouldn't be able to find them in the conda prefix I think.

I did not mean to delete these things but they were deleted before and not fully deleting them caused trouble because of the unresolved symlinks in there.

Reply:
This is now https://github.com/conda-forge/gap-feedstock/issues/70